api/api.py

```diff
@@ -270,7 +271,13 @@ def chunked_completion(self, completion_request: CompletionRequest):
         )
         generator_args = GeneratorArgs(
             completion_request.messages[-1].get("content"),
+            max_new_tokens=(
+                int(completion_request.max_tokens)
+                if completion_request.max_tokens
+                else OPENAI_API_DEFAULT_MAX_TOKENS
+            ),
             encoded_prompt=encoded,
+            temperature=float(completion_request.temperature),
```

Comment:
ditto

Reply:
See previous re: typecasting for dataclasses.